modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -2,67 +2,54 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../../assets/images/
-
-Configuration and usage information.
 
+Read this topic to obtain additional configuration and usage information for the Anypoint Connector for Salesforce (Salesforce Connector).
 
 [[customevents]]
 == Custom Event Notifications
 
-The Salesforce connector provides two operations that are useful for getting custom event notifications. These notifications pertain to general events that are not tied to Salesforce data changes.
+Salesforce Connector enables you to obtain custom event notifications. These notifications apply to general events that are not tied to Salesforce data changes.
+
+To obtain custom event notifications:
 
-. Create a streaming channel with the Publish Streaming Channel operation.
+. Create a streaming channel by using the `Publish Streaming Channel` operation.
 +
 A StreamingChannel is a special Salesforce object that represents a channel used for notifying listeners of generic Streaming API events.
 +
-Note that you can also create a streaming channel through the Salesforce or through Workbench.
+You can also create a streaming channel through Salesforce or https://workbench.developerforce.com/about.php[
+workbench].
 +
-. Subscribe to the channel through the `Subscribe Channel Listener` operation.
+. Subscribe to the channel by using the `Subscribe Channel Listener` operation.
 +
-The Salesforce connector converts the custom events in your streaming channel to Mule events and dispatches them to your flows.
+Salesforce Connector converts the custom events in your streaming channel to Mule events and sends them to your flows.
+
+For more information about working with streaming channels, see <<streamingchanel,Create a Streaming Channel to Receive Data From Salesforce>> and <<receivedata,Receive Data From Salesforce>>.
 
 [[eventsandtopics]]
 == Events and Topics
 
-Your application can receive events by subscribing to a Salesforce topic.
-
-Each event that travels through your flows contains information about the Salesforce data that has changed, how it changes, and when. The connector parses this information and sends you information that a flow can work with.
-
-Inbound properties of events:
+Your app can receive events by subscribing to an existing Salesforce topic or by creating a new topic and then subscribing to it.
 
-* `payload`
-* `createdDate`
-* `replayId`
+Each event that travels through your flow contains information about the Salesforce data that changed, including how the data changed and when the change occurred. The connector parses this information and sends your app the information.
```

Comment:
I think here the phrasing is not right. _The connector parses this information and sends your app the information._ can be removed.